lib/internal/url.js

```diff
@@ -534,15 +533,27 @@ ObjectDefineProperties(URLSearchParams.prototype, {
   },
 });
 
+/**
+ * Checks if a value has the shape of a WHATWG URL object.
+ *
+ * Using a symbol or instanceof would not be able to recognize URL objects
+ * coming from other implementations (e.g. in Electron), so instead we are
+ * checking some well known properties for a lack of a better test.
+ *
+ * @param {*} self
+ * @returns {asserts self is URL}
```

Comment:
Sorry again, `asserts` would mean that the function throws, that's not what we're doing here.
```suggestion
 * @returns {self is URL}
```

Reply:
No worries! Updated the pull request. Thanks.